javav2/usecases/creating_scheduled_events/Readme.md

```diff
@@ -0,0 +1,536 @@
+#  Creating scheduled events to invoke Lambda functions
+
+You can create a scheduled event that invokes an AWS Lambda function by using an Amazon CloudWatch Event. You can configure a CloudWatch Event to use a cron expression to schedule when a Lambda function is invoked. For example, you can schedule a CloudWatch Event to invoke an AWS Lambda function every weekday. AWS Lambda is a compute service that enables you to run code without provisioning or managing servers.
+
+**Note**: You can create Lambda functions in various programming languages. For this tutorial, the AWS Lambda function is implemented by using the AWS Lambda Java runtime API. For more information about AWS Lambda, see
+[What is AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html).
+
+In this tutorial, you create a Lambda function by using the AWS Lambda Java runtime API. This example invokes different AWS services to perform a specific use case. For example, assume that an organization sends a mobile text message to its employees that congratulates them at the one year anniversary date, as shown in this illustration.
+
+![AWS Tracking Application](images/picPhone.png)
+
+This tutorial shows you how to use Java logic to create a solution that performs this use case. For example, you'll learn how to read an Amazon DynamoDB table to determine which employees have reached the one year anniversary date, how to process the data, and send out a text message all by using an AWS Lambda function. Then you’ll learn how to use a cron expression to invoke the AWS Lambda function every weekday. 
+
+This AWS tutorial uses an Amazon DynamoDB table named **Employee** that contains these fields. 
+-	**Id** – the key for the table.
+-	**first** – the employee’s first name.
+-	**phone** – the employee’s phone number.
+-	**startDate** – the employee’s start date.
+
+![AWS Tracking Application](images/pic00.png)
+
+**Cost to complete**: The AWS services included in this document are included in the [AWS Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc).
+
+**Note**: Be sure to terminate all of the resources after you have completed this tutorial to ensure that you are not charged.
+
+#### Topics
++	Prerequisites
++	Create an IAM role that is used to execute Lambda functions
```

Comment:
"IAM" -> "AWS Identity and Access Management (IAM)" on first mention in a code file.